src/System.Net.WebSockets.Client/src/System.Net.WebSockets.Client.csproj

```diff
@@ -6,13 +6,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
-    <Configuration Condition=" '$(Configuration)' == '' ">Windows_Debug</Configuration>
```

Comment:
The default configuration must be set before the import of dir.props, which it still is if you look a few lines up :)